### PR TITLE
Buildbot mabr inband mani init transport

### PR DESCRIPTION
**Pull request by soheibthriber**
DVB mabr: Adding an option to offer the possibility of selecting how the manifests and init
segments are delivered:
         - either in the multicast gateway configuration transport session .
         - or in the multicast transport sessions delivering the content.

### CODE REVIEW

File: src/filters/out_route.c [hunk 1]
```diff
@@ -2117,11 +2170,16 @@ void routeout_send_fdt(GF_ROUTEOutCtx *ctx, ROUTEService *serv, ROUTEPid *rpid)
 	gf_dynstrcat(&payload, "<?xml version=\"1.0\" encoding=\"UTF-8\" standalone=\"no\" ?>\n", NULL);
 	gf_dynstrcat(&payload, "<FDT-Instance Expires=\"3916741152\" xmlns=\"urn:IETF:metadata:2005:FLUTE:FDT\">\n", NULL);
 	//todo: inject manifest, init seg and child playlist ?
```
Comment (jeanlf) [inline src/filters/out_route.c:2172]: can you remove the todo comment now that this is done ?

File: src/filters/out_route.c [hunk 2]
```diff
@@ -2999,6 +3067,7 @@ static const GF_FilterArgs ROUTEOutArgs[] =
 		"- all: send checksum for everything", GF_PROP_UINT, "meta", "no|meta|all", 0},
 	{ OFFS(recv_obj_timeout), "set timeout period in ms before client resorts to unicast repair", GF_PROP_UINT, "50", NULL, 0},
 	{ OFFS(errsim), "simulate errors using a 2-state Markov chain. Value are percentages", GF_PROP_VEC2, "0.0x100.0", NULL, 0},
+	{ OFFS(flute_inband_mani_init), "DVB mabr option: If true send the mani and init segment in content transport sessions instead of configuration transport session", GF_PROP_BOOL, "false", NULL, 0},
```
Comment (jeanlf) [inline src/filters/out_route.c:3070]: Can you please rename to `use_inband` ?

**Review by jeanlf — CHANGES_REQUESTED** [review]

File: src/filters/out_route.c (hunk 2 above)
Comment (soheibthriber) [reply src/filters/out_route.c]: Done

File: src/filters/out_route.c (hunk 1 above)
Comment (soheibthriber) [reply src/filters/out_route.c]: Done

**Review by rbouqueau — APPROVED** [review]

**Comment by rbouqueau** [comment]
LGTM: merging

**Pull request merged**